script.module.pyhdhr/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyhdhr" version="1.0.2" name="pyHDHR" provider-name="Lunatixz">
```

Comment:
Just wondering is this your module or was it just packaged as a script module? Googling pyhdhr gives a few results so that's why i am asking. If the module is just made by someone else you have to keep the original author. If it is inspired by please check the licence.